app/src/main/res/layout/event_layout.xml

```diff
@@ -44,39 +44,53 @@
                 app:circleImageUrl="@{ event.thumbnailImageUrl }"
                 app:placeholder="@{ @drawable/ic_photo_shutter }"/>
 
-            <TextView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@{event.name}"
-                android:textSize="@dimen/text_size_normal"
-                android:textColor="@android:color/black"
-                android:paddingEnd="@dimen/spacing_medium"
-                android:paddingLeft="@dimen/spacing_medium"
-                android:paddingRight="@dimen/spacing_medium"
-                android:paddingStart="@dimen/spacing_medium"/>
-
-            <FrameLayout
+            <RelativeLayout
```

Comment:
RelativeLayout is not needed for this job. What feature is it providing that a linearlayout may not

Reply:
If you require LinearLayout then I'll change it. It's just that Relative Layout does only one extra measure than LinearLayout and since the hierarchy isn't much complex so it would not cause any performance related issue. Anyways no problem, I'll change it.

Reply:
Actually we'll later want to use constraint layout, and it is easier to migrate this way

Reply:
@iamareebjamal  Currently, Constraint Layouts are used in very few layout files. Should I create an issue for migrating the existing layouts to constraint layouts?